modules/admin-ui/src/main/java/org/opencastproject/adminui/endpoint/SeriesEndpoint.java

```diff
@@ -497,6 +510,52 @@ public Response getNewThemes() {
     return Response.ok(themesJson.toJSONString()).build();
   }
 
+  @GET
+  @Path("new/tobira/page")
+  @Produces(MediaType.APPLICATION_JSON)
+  @RestQuery(
+          name = "getTobiraPage",
+          description = "Returns data about the page tree of a connected Tobira instance for use in the series creation wizard",
+          returnDescription = "Information about a given page in Tobira, and its direct children",
+          restParameters = { @RestParameter(
+                          name = "path",
+                          isRequired = true,
+                          type = STRING,
+                          description = "The path of the page you want information about"
+                  ) },
+          responses = {
+                  @RestResponse(
+                          responseCode = SC_OK,
+                          description = "Data about the given page in Tobira. Note that this does not mean the page exists!"),
+                  @RestResponse(
+                          responseCode = SC_NOT_FOUND,
+                          description = "Nonexistent `path`"),
+                  @RestResponse(
+                          responseCode = SC_BAD_REQUEST,
+                          description = "missing `path`"),
+                  @RestResponse(
+                          responseCode = SC_SERVICE_UNAVAILABLE,
+                          description = "Tobira is not configured (correctly)") })
```

Comment:
Weird indentation. At least the newly formatted files in Opencast would use this formatting:

```java
@RestQuery(
    name = "getTobiraPage",
    description = "Returns data about the page tree of a connected Tobira instance for use "
        + "in the series creation wizard",
    returnDescription = "Information about a given page in Tobira, and its direct children",
    restParameters = { 
        @RestParameter(
            name = "path",
            isRequired = true,
            type = STRING,
            description = "The path of the page you want information about"
        ),
    },
    responses = {
        @RestResponse(
            responseCode = SC_OK,
            description = "Data about the given page in Tobira. Note that this does not mean the page exists!"
        ),
        @RestResponse(
            responseCode = SC_NOT_FOUND,
            description = "Nonexistent `path`"
        ),
        @RestResponse(
            responseCode = SC_BAD_REQUEST,
            description = "missing `path`"
        ),
        @RestResponse(
            responseCode = SC_SERVICE_UNAVAILABLE,
            description = "Tobira is not configured (correctly)"
        ) 
    }
)
```

Now, the admin UI code was not yet formatted (bc it contained by far the most incorrectly formatted code). If you want to be consistent with the existing admin UI code, we can leave it like that. But ye, dunno, I don't particularly like it. And given that we will format everything at one point, it will change anyway?

Reply:
Because of everything you said, and also because this already took longer than we planned, let's just leave it?